src/createNativeStackNavigator.js

```diff
@@ -14,7 +14,9 @@ import {
   SceneView,
   StackActions,
   StackRouter,
+  // eslint-disable-next-line import/no-unresolved
```

Comment:
Why did you ignore this line? If you use react-navigation why you don't depend on it in package.json?